source/state/swap/api.ts

```diff
@@ -0,0 +1,181 @@
+/////////////////////////
+// Imports
+/////////////////////////
+
+import { createAsyncThunk } from "@reduxjs/toolkit";
+
+/////////////////////////
+// Types
+/////////////////////////
+
+import {
+  IStageTransaction,
+  ISearchResponse,
+  ISearchCurrency,
+  ICurrencyRate,
+  IPendingTransaction,
+  IExolixTransaction
+} from "./types";
+import { RootState } from 'state/store';
+
+
+/////////////////////////
+// Constants
+/////////////////////////
+
+const API_KEY = 'Zm5w8D384z4E9Yma24Adn24obD';
```

Comment:
This API_KEY is already stored in the `.env` file. We shouldn't expose this value here.